### PR TITLE
Giphy: Add attribution image to no results view

### PR DESCRIPTION
**Pull request by frosty**
As part of our Giphy integration, we need to add a Giphy attribution image to the app. I've added this to the no results view in the Giphy media picker, by adding a new `subtitleImage` view to `NoResultsViewController`. I added a separate initialiser so that a `NoResultsViewController` has _either_ subtitle labels or a subtitle image view.

simulator screen shot - iphone 8 - 2018-09-05 at 21 11 39

**To test**

* Build and run
* Navigate to the media library for a site and tap the + button in the upper right
* Choose Giphy from the action sheet
* Check you can see the attribution logo in the no results view

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift [hunk 1]
```diff
@@ -124,6 +149,27 @@ import UIKit
...
+    /// Public method to provide values for text elements.
+    ///
+    /// - Parameters:
+    ///   - title:              Main descriptive text. Required.
+    ///   - buttonTitle:        Title of action button. Optional.
+    ///   - image:              Name of image file to use. Optional.
+    ///   - subtitleImage:      Name of image file to use in place of subtitle. Optional.
+    ///   - accessoryView:      View to show instead of the image. Optional.
+    ///
+    @objc func configure(title: String,
```
[the remark this answers is not in the document]
Comment (frosty) [reply WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: My thought here was so we’re only supporting _either_ subtitle text _or_ an image. Although I suppose there’s no reason we couldn’t allow both. What do you think?

File: WordPress/Classes/ViewRelated/Media/Giphy/NoResultsGiphyConfiguration.swift [hunk 2]
```diff
@@ -5,9 +5,8 @@ struct NoResultsGiphyConfiguration {
     static func configureAsIntro(_ viewController: NoResultsViewController) {
         viewController.configure(title: .giphyPlaceholderTitle,
```
[the remark this answers is not in the document]
Comment (frosty) [reply WordPress/Classes/ViewRelated/Media/Giphy/NoResultsGiphyConfiguration.swift]: Whoops, thanks - totally missed those!

File: WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: Oh I wasn't proposing we support both. Either text or image is fine with me. 

My thought was there is already an either/or paradigm. Ex you can either have an `accessoryView` or `imageView`, but not both; you can have either `subtitleText` or `attributedSubtitleText` but not both. And you adopted this with subtitle text or image.

It just seemed a bit confusing to me to have to determine which `configure` method to use for what situation, when, if we had one `configure` method, all one would have to do is set the needed values.

[the remark this answers is not in the document]
Comment (frosty) [reply WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: Okay, sure :) Happy to change this to one.

**Comment by frosty** [comment]
@ScoutHarris Thanks for the review! Ready for another round!

**Comment by ScoutHarris** [comment]
Thanks @frosty ! :shipit:

**Pull request merged**